lib/puppet_litmus/rake_tasks.rb

```diff
@@ -8,6 +8,7 @@
 require 'json'
 require 'parallel'
 require 'tty-spinner'
+require 'pdk/util'
```

Comment:
I see this is lazy loaded in this PR: https://github.com/puppetlabs/puppetlabs_spec_helper/pull/295/files

Should we move this load down?